request.proto

```diff
@@ -283,3 +284,12 @@ message EquipmentReportsRequest {
     optional string filter          = 4;
     repeated string forbidden_uris  = 5;
 }
+
+message VehiclePositionsRequest {
+    optional int32 depth            	= 1;
+    optional int32 start_page       	= 2;
+    optional int32 count            	= 3;
+    optional string filter          	= 4;
+    repeated string forbidden_uris  	= 5;
+    optional uint64 _current_datetime	= 6;
```

Comment:
This is already present in https://github.com/CanalTP/navitia-proto/blob/master/request.proto#L223

Reply:
:+1: